### DISCUSSION TITLE
Where is .profiles/default/auth.json ?

### DISCUSSION THREAD

**Discussion by charlessxavier** [UltimaHoarder/UltimaScraper#1401] [Q&A]
hi, I did follow all the steps as mentioned, after running the start_ofd.py all i got was a message to edit config.json

but there is no folder with the name .profile for me to edit the auth.json? Please help..

**Comment by fightcancer2** [comment]
After you run start_ofd.py the first time, it will prompt you to update your config.json file.  Then it creates the .profiles/default/auth.json file.  Then you can edit it.

**Reply by havingpun** [reply]
After running that .py it only creates a .settings location. There's no .profiles still

nevermind fixed it by reinstalling python

**Comment by lucky1luc** [comment]
Maybe you should make hidden folders visible?

**Comment by Texxer** [comment]
My issue is even worse. Reinstalling Python didn't help. Basically, I Press enter to continue and then nothing happens.

![Code_oQQmM4Ku6g](https://user-images.githubusercontent.com/22458550/190275429-a3f9e94d-f4c1-4776-aaac-a79bbc9915ba.png)
